app/controllers/registers_controller.rb

```diff
@@ -18,7 +18,7 @@ def create
     end
 
     render json: {response: response}
-    return 
+    return
```

Comment:
:heart: for whitespace cleanup. Although we probably don't even need the `return` here btw.